examples/query/react/basic/src/App.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { rest } from 'msw'
```

Comment:
As for the location of the `App.test.tsx` file, let me know what y'all would prefer I do. We can do the `__test__` directory in each directory where a file is present, usually I just put the test file right next to the component being tested for clickability.

Reply:
Right next to it is a lot less confusing I think.